Source/WebGPU/WGSL/AST/ASTBuiltinAttribute.h

```diff
@@ -33,17 +33,17 @@ class BuiltinAttribute final : public Attribute {
     WTF_MAKE_FAST_ALLOCATED;
 
 public:
-    BuiltinAttribute(SourceSpan span, StringView name)
+    BuiltinAttribute(SourceSpan span, String name)
```

Comment:
You're doing a double copy here; surely you want the parameter to be `const String&`? Or you could keep the parameter a `StringView` but still change the parameter to `String m_name` like you're doing below.

Reply:
That's fair, I didn't worry too much since the underlying string isn't being copied anyway, so it's equivalent to the StringView code (right?). But in any case, no downside in making it `const String&`.